workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -0,0 +1,87 @@
+
+using System.Collections.Generic;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+
+namespace Improbable.Gdk.Core.Commands
+{
+    public class CreateEntity
+    {
+        public struct Request
+        {
+            public Worker.Core.Entity Entity;
+            public EntityId? EntityId;
+            public uint? TimeoutMillis;
+            public uint SenderEntityId;
+
+            internal RequestId<CreateEntityRequest> Send(Connection connection)
+            {
+                return connection.SendCreateEntityRequest(Entity, EntityId, TimeoutMillis);
```

Comment:
Do nullables and `Option` actually interop nicely? Amazing.

Reply:
The C# Core SDK has no `Improbable.Option` its all nullables 😉

Reply:
Effing amazeballs.